### DISCUSSION TITLE
NACA multipoint example with runScript_v2.py is broken - any ideas how to fix?

### DISCUSSION THREAD

**Discussion by twuerfl** [mdolab/dafoam#618] [Bug]
Output Message:

+------------------------------------------------------------------------------+
| pyDAFoam Error: Primal solution failed for the baseline design!            |
+------------------------------------------------------------------------------+

Traceback (most recent call last):
  File "/home/dafoamuser/dafoam/packages/miniconda3/lib/python3.8/site-packages/pyoptsparse/pyIPOPT/pyIPOPT.py", line 219, in eval_g
    fcon, fail = self._masterFunc(x, ["fcon"])
  File "/home/dafoamuser/dafoam/packages/miniconda3/lib/python3.8/site-packages/pyoptsparse/pyOpt_optimizer.py", line 320, in _masterFunc
    result = self._masterFunc2(*args)
  File "/home/dafoamuser/dafoam/packages/miniconda3/lib/python3.8/site-packages/pyoptsparse/pyOpt_optimizer.py", line 403, in _masterFunc2
    args = self.optProb.objFun(xuser)
  File "/home/dafoamuser/dafoam/packages/miniconda3/lib/python3.8/site-packages/dafoam/optFuncs.py", line 117, in calcObjFuncValuesMP
    DASolver.evalFunctions(funcs, evalFuncs=evalFuncs)
  File "/home/dafoamuser/dafoam/packages/miniconda3/lib/python3.8/site-packages/dafoam/pyDAFoam.py", line 1413, in evalFunctions
    raise Error("Primal solution failed for the baseline design!")
dafoam.pyDAFoam.Error

Number of Iterations....: 0

Number of objective function evaluations             = 0
Number of objective gradient evaluations             = 0
Number of equality constraint evaluations            = 0
Number of inequality constraint evaluations          = 1
Number of equality constraint Jacobian evaluations   = 0
Number of inequality constraint Jacobian evaluations = 0
Number of Lagrangian Hessian evaluations             = 0
Total CPU secs in IPOPT (w/o function evaluations)   =      0.000
Total CPU secs in NLP function evaluations           =      0.081

EXIT: Invalid number in NLP function or derivative detected.

Optimization Problem -- opt
================================================================================
    Objective Function: calcObjFuncValuesMP

    Solution: 
--------------------------------------------------------------------------------
    Total Time:                    0.0826
       User Objective Time :       0.0000
       User Sensitivity Time :     0.0000
       Interface Time :            0.0000
       Opt Solver Time:            0.0826
    Calls to Objective Function :       0
    Calls to Sens Function :            0

   Objectives
      Index  Name            Value
          0  obj     0.000000E+00

   Variables (c - continuous, i - integer, d - discrete)
      Index  Name          Type      Lower Bound            Value      Upper Bound     Status
          0  mp0_alpha_0      c     0.000000E+00     3.008097E+00     1.000000E+01           
          1  mp1_alpha_0      c     0.000000E+00     0.000000E+00     1.000000E+01          l
          2  mp2_alpha_0      c     0.000000E+00     7.622412E+00     1.000000E+01           
          3  shapey_0         c    -1.000000E+00     0.000000E+00     1.000000E+00           
          4  shapey_1         c    -1.000000E+00     1.000000E+00     1.000000E+00          u
          5  shapey_2         c    -1.000000E+00     0.000000E+00     1.000000E+00           
          6  shapey_3         c    -1.000000E+00     0.000000E+00     1.000000E+00           
          7  shapey_4         c    -1.000000E+00     0.000000E+00     1.000000E+00           
          8  shapey_5         c    -1.000000E+00     0.000000E+00     1.000000E+00           
          9  shapey_6         c    -1.000000E+00     0.000000E+00     1.000000E+00           
         10  shapey_7         c    -1.000000E+00     0.000000E+00     1.000000E+00           
         11  shapey_8         c    -1.000000E+00     0.000000E+00     1.000000E+00           
         12  shapey_9         c    -1.000000E+00     0.000000E+00     1.000000E+00           
         13  shapey_10        c    -1.000000E+00     0.000000E+00     1.000000E+00           
         14  shapey_11        c    -1.000000E+00     0.000000E+00     1.000000E+00           
         15  shapey_12        c    -1.000000E+00     0.000000E+00     1.000000E+00           
         16  shapey_13        c    -1.000000E+00     0.000000E+00     1.000000E+00           
         17  shapey_14        c    -1.000000E+00     0.000000E+00     1.000000E+00           
         18  shapey_15        c    -1.000000E+00     0.000000E+00     1.000000E+00           
         19  shapey_16        c    -1.000000E+00     0.000000E+00     1.000000E+00           
         20  shapey_17        c    -1.000000E+00     0.000000E+00     1.000000E+00           
         21  shapey_18        c    -1.000000E+00     0.000000E+00     1.000000E+00           
         22  shapey_19        c    -1.000000E+00     0.000000E+00     1.000000E+00           

   Constraints (i - inequality, e - equality)
      Index  Name                              Type          Lower           Value           Upper    Status  Lagrange Multiplier (N/A)
          0  DVCon1_volume_constraint_0           i   1.000000E+00    0.000000E+00    3.000000E+00         L    9.00000E+100
          1  DVCon1_thickness_constraints_0       i   8.000000E-01    0.000000E+00    3.000000E+00         L    9.00000E+100
          2  DVCon1_thickness_constraints_0       i   8.000000E-01    0.000000E+00    3.000000E+00         L    9.00000E+100
          3  DVCon1_thickness_constraints_0       i   8.000000E-01    0.000000E+00    3.000000E+00         L    9.00000E+100
          4  DVCon1_thickness_constraints_0       i   8.000000E-01    0.000000E+00    3.000000E+00         L    9.00000E+100
          5  DVCon1_thickness_constraints_0       i   8.000000E-01    0.000000E+00    3.000000E+00         L    9.00000E+100
          6  DVCon1_thickness_constraints_0       i   8.000000E-01    0.000000E+00    3.000000E+00         L    9.00000E+100
          7  DVCon1_thickness_constraints_0       i   8.000000E-01    0.000000E+00    3.000000E+00         L    9.00000E+100
          8  DVCon1_thickness_constraints_0       i   8.000000E-01    0.000000E+00    3.000000E+00         L    9.00000E+100
          9  DVCon1_thickness_constraints_0       i   8.000000E-01    0.000000E+00    3.000000E+00         L    9.00000E+100
         10  DVCon1_thickness_constraints_0       i   8.000000E-01    0.000000E+00    3.000000E+00         L    9.00000E+100
         11  DVCon1_thickness_constraints_0       i   8.000000E-01    0.000000E+00    3.000000E+00         L    9.00000E+100
         12  DVCon1_thickness_constraints_0       i   8.000000E-01    0.000000E+00    3.000000E+00         L    9.00000E+100
         13  DVCon1_thickness_constraints_0       i   8.000000E-01    0.000000E+00    3.000000E+00         L    9.00000E+100
         14  DVCon1_thickness_constraints_0       i   8.000000E-01    0.000000E+00    3.000000E+00         L    9.00000E+100
         15  DVCon1_thickness_constraints_0       i   8.000000E-01    0.000000E+00    3.000000E+00         L    9.00000E+100
         16  DVCon1_thickness_constraints_0       i   8.000000E-01    0.000000E+00    3.000000E+00         L    9.00000E+100
         17  DVCon1_thickness_constraints_0       i   8.000000E-01    0.000000E+00    3.000000E+00         L    9.00000E+100
         18  DVCon1_thickness_constraints_0       i   8.000000E-01    0.000000E+00    3.000000E+00         L    9.00000E+100
         19  DVCon1_thickness_constraints_0       i   8.000000E-01    0.000000E+00    3.000000E+00         L    9.00000E+100
         20  DVCon1_thickness_constraints_0       i   8.000000E-01    0.000000E+00    3.000000E+00         L    9.00000E+100
         21  DVCon1_linear_constraint_0_shapey    e   0.000000E+00    0.000000E+00    0.000000E+00              9.00000E+100
         22  DVCon1_linear_constraint_0_shapey    e   0.000000E+00    0.000000E+00    0.000000E+00              9.00000E+100
         23  DVCon1_linear_constraint_0_shapey    e   0.000000E+00    0.000000E+00    0.000000E+00              9.00000E+100
         24  DVCon1_linear_constraint_0_shapey    e   0.000000E+00    0.000000E+00    0.000000E+00              9.00000E+100
         25  DVCon1_linear_constraint_0_shapey    e   0.000000E+00    0.000000E+00    0.000000E+00              9.00000E+100
         26  DVCon1_linear_constraint_0_shapey    e   0.000000E+00    0.000000E+00    0.000000E+00              9.00000E+100
         27  DVCon1_linear_constraint_0_shapey    e   0.000000E+00    0.000000E+00    0.000000E+00              9.00000E+100
         28  DVCon1_linear_constraint_0_shapey    e   0.000000E+00    0.000000E+00    0.000000E+00              9.00000E+100
         29  DVCon1_linear_constraint_0_shapey    e   0.000000E+00    0.000000E+00    0.000000E+00              9.00000E+100
         30  DVCon1_linear_constraint_0_shapey    e   0.000000E+00    0.000000E+00    0.000000E+00              9.00000E+100
         31  DVCon1_linear_constraint_1_shapey    e   0.000000E+00    0.000000E+00    0.000000E+00              9.00000E+100
         32  DVCon1_linear_constraint_1_shapey    e   0.000000E+00    0.000000E+00    0.000000E+00              9.00000E+100
         33  mp0_CL                               e   3.000000E-01    0.000000E+00    3.000000E-01         E    9.00000E+100
         34  mp1_CL                               e   7.000000E-01    0.000000E+00    7.000000E-01         E    9.00000E+100
         35  mp2_CL                               e   5.000000E-01    0.000000E+00    5.000000E-01         E    9.00000E+100

   Exit Status
      Inform  Description
         -13  Invalid Number Detected
--------------------------------------------------------------------------------

**Comment by friedenhe** [comment]
Thanks for the feedback. We reproduced this issue and are fixing it. On a separate note, the v2 multipoint capability is not being maintained, please use the v3 script: runScript.py
